Changelog.md

```diff
@@ -1,5 +1,26 @@
 # @kineticcafe/result
 
+## Unreleased
+
+- Renamed `unwrapErrors` to `unwrapErrs`, `unwrapAndThrowErrors` to `throwErrs`,
+  and `unwrapResults` to `unwrapOks`. Despite the suggestion inherent in the
+  name, an `Err` is not _necessarily_ an `Error`. (In Haskell, this is the
+  `Either` type with `Left` and `Right` branches.) The old names are deprecated
```

Comment:
This describes what the Haskell thing is, not why. It is used by Haskell to formalize and streamline error handling, so it seems to me its name reflects its intended use rather than its contents.